package trex.hackathon.smart_prep.exception;

public class QuizAttemptNotFoundException extends RuntimeException {
	public QuizAttemptNotFoundException(String message) {
		super(message);
	}
}